packages/angular_devkit/core/node/host_spec.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+// tslint:disable:no-any
+// tslint:disable:non-null-operator
+import { normalize, virtualFs } from '@angular-devkit/core';
+import { NodeJsAsyncHost } from '@angular-devkit/core/node';
+import * as fs from 'fs';
+import { Observable } from 'rxjs/Observable';
+import { Subscription } from 'rxjs/Subscription';
+
+const temp = require('temp');
+
+
+describe('NodeJsAsyncHost', () => {
+  let root: string;
+  let host: virtualFs.Host<fs.Stats>;
+
+  beforeEach(() => {
+    root = temp.mkdirSync('core-node-spec-');
+    host = new virtualFs.ScopedHost(new NodeJsAsyncHost(), normalize(root));
+  });
+  afterEach(done => {
+    host.delete(normalize('/'))
+      .subscribe({ complete() { done(); } });
+  });
+
+  it('can watch', done => {
```

Comment:
Not a blocker for this PR but we'll need tests for the other methods. I can do those later.